src/component/EditText.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+
+function EditText() {
+  const [create, setCreate] = useState('');
+
+  function handleClick(e) {
+    e.preventDefault();
+    setCreate(e.target.value);
+  }
+
+  return (
+    <>
+      <input
+        type="text"
+        name="edit"
+        value={create}
+        onChange={handleClick}
+      ></input>
+      <button
+      // onClick={() => setSave([...save, create])}
```

Comment:
Should this be commented?

Reply:
sorry,I forgot to delete this.I was trying something